README.md

```diff
@@ -3,6 +3,7 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubeedge/kubeedge)](https://goreportcard.com/report/github.com/kubeedge/kubeedge)
 [![LICENSE](https://img.shields.io/github/license/kubeedge/kubeedge.svg?style=flat-square)](https://github.com/kubeedge/kubeedge/blob/master/LICENSE)
 [![Releases](https://img.shields.io/github/release/kubeedge/kubeedge/all.svg?style=flat-square)](https://github.com/kubeedge/kubeedge/releases)
+[![Documentation Status](https://readthedocs.org/projects/kubeedge-anvith-sphinx/badge/?version=latest)](https://kubeedge-anvith-sphinx.readthedocs.io/en/latest/?badge=latest)
```

Comment:
this url should not have author name in it right ?

Reply:
Will remove this for now. Will add the documentation status badge after we have the documentation live.

Reply:
Fixed this. Removed the line.